projects/packages/sync/src/replicastore/class-table-checksum.php

```diff
@@ -456,7 +456,8 @@ protected function prepare_filter_values_as_sql( $filter_values = array(), $tabl
 			switch ( $filter['operator'] ) {
 				case 'IN':
 				case 'NOT IN':
-					$values_placeholders = implode( ',', array_fill( 0, count( $filter['values'] ), '%s' ) );
+					$filter_values_count = is_countable( $filter['values'] ) ? count( $filter['values'] ) : 0;
```

Comment:
This is probably less correct than the previous code. Really there should be some explicit error handling here for the case that `$filter['values']` is not an array or is an empty array.

Back in PHP 7 the old code would have unintentionally worked for non-null scalars, only producing invalid SQL when passed null. Now it'll produce invalid SQL for any non-countable.